EasyInventory/src/main/java/com/easyinventory/enterprise/controller/EasyInventoryController.java

```diff
@@ -61,13 +72,20 @@
 
     //Endpoints for the Items
 
-    //Retrieves all the items from the database
+    /**
+     * Retrieves all the items from the database
+     * @return
+     */
     @RequestMapping(value="/item/all", method = RequestMethod.GET)
     public List<Item> getAllItems(){
         return itemRepository.findAllByOrderByCategoryIdAsc();
     }
 
-    //Adds a new value to the item table
+    /**
+     * Adds a new value to the item table
+     * @param item
+     * @return
```

Comment:
When making JavaDoc, complete it.  Specify what the parameters and return values are.  Otherwise, this is incomplete and does not offer as much value as it could.